RELEASE.md

```diff
@@ -133,11 +133,17 @@ To release a new version of Spyder you need to follow these steps:
 * Update version in `__init__.py` (set release version, remove 'dev0', add 'preX'), then
 
       git add .
-      git commit -m "Release X.X.XpreX"
-      git tag vX.X.XpreX
-      git push upstream --tags
+      git commit -m "Release X.X.XpreX [ci skip]"
+      git push upstream 5.x
 
-* If workflows succeed, proceed to steps to publish the release; otherwise merge a fix PR and repeat previous steps with incremented 'preX'
+* Manually activate the workflows:
+    - Create Windows Installer
+    - Create macOS App Bundle and DMG
+    - Create conda-based installers for Windows, macOS, and Linux
```

Comment:
I don't understand this part: how is the release manager supposed to activate these workflows?

Reply:
The release manager will have to go to the Actions tab in the GitHub repository web interface, select the desired workflow, click the "Run workflow" button, and select the 5.x branch in the popup window. Then repeat for the remaining workflows. See [Running a workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow).

Note that the "Run workflow" button is not available until the updated workflow file is merged into `master` branch, so is not currently visible. I've tested this on my fork by merging it into `master` there. Should I include the [Running a workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow#running-a-workflow) link?

Reply:
I think having some reference info here regarding how to manually trigger the workflows could be nice :+1:

Reply:
I will reference the instructions for manually running a workflow